assets/scss/base/_helpers.scss

```diff
@@ -177,3 +177,18 @@
 	@extend %no-bullet;
 }
 
+%break-word{
+    -ms-word-break: break-all;
+        word-break: break-all;
+
+        word-break: break-word;
```

Comment:
It's for compatibility. See here - https://css-tricks.com/almanac/properties/w/word-break/